s3-sqs-lambda-terraform/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "AWS Amazon S3 Lambda with SQS queue",
+  "description": "Create a Lambda function triggered for every jpg image file stored to S3 via SQS.",
+  "language": "node.js",
+  "level": "200",
+  "framework": "Terraform",
+  "introBox": {
+    "headline": "Create a Lambda function triggered for every jpg image file stored to S3 via SQS",
+    "text": [
+      "The terraform manifest deploys a Lambda function, an SQS queue, one S3 bucket and the IAM resources required to run the application.",
+      "An SQS queue consumes ObjectCreated events from an Amazon S3 bucket if the file has .jpg extension. The SQS triggers a Lambda function.",
+      "The Lambda function serve as a pre-configured template, providing a starting point for developing your application."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-sqs-lambda-terraform",
+      "templateURL": "serverless-patterns/s3-sqs-lambda-terraform",
+      "projectFolder": "s3-sqs-lambda-terraform",
+      "templateFile": "main.tf"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Configuring an Amazon SQS queue to trigger an AWS Lambda function",
+        "link": "https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-configure-lambda-function-trigger.html",
+      },
+      {
+        "text": "Amazon Simple Storage Service (S3) -> User Guide -> Using EventBridge",
```

Comment:
How does EventBridge relate to the rest of the pattern? It isn't mentioned anywhere else

Reply:
Right and sorry for that. Reference deleted.